packages/server/lib/screenshots.ts

```diff
@@ -32,20 +33,35 @@ const MIN_PREFIX_BYTES = 64
 
 // when debugging logs automatically prefix the
 // screenshot id to the debug logs for easier association
+// @ts-ignore
 debug = _.wrap(debug, (fn, str, ...args) => {
   return fn(`(${__ID__}) ${str}`, ...args)
 })
```

Comment:
I think you can get rid of the @ts-ignore with this
```suggestion
}) as Debug.Debugger
```

Reply:
[`7d6e3a0` (#30758)](https://github.com/cypress-io/cypress/pull/30758/commits/7d6e3a0b960a487f596937ecb85371a3e39534eb)